microsoft/testsuites/network/sriov.py

```diff
@@ -34,26 +34,36 @@
 )
 class Sriov(TestSuite):
     TIME_OUT = 300
+    # ConnectX-3 uses mlx4_core
```

Comment:
Explain, why the dict is needed here. The reason is like if the dependent module is not removed, the module like mlx5_core cannot be unloaded with error "in use"...

Reply:
updated